src/user-event/press/__tests__/press.real-timers.test.tsx

```diff
@@ -0,0 +1,304 @@
+import * as React from 'react';
+import {
+  Pressable,
+  Text,
+  TextInput,
+  TouchableHighlight,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { createEventLogger, getEventsName } from '../../../test-utils';
+import { render, screen } from '../../..';
+import { userEvent } from '../..';
+import * as WarnAboutRealTimers from '../utils/warnAboutRealTimers';
+
+const mockWarnAboutRealTimers = jest
+  .spyOn(WarnAboutRealTimers, 'warnAboutRealTimers')
+  .mockImplementation();
+
+describe('userEvent.press with real timers', () => {
+  beforeEach(() => {
+    jest.useRealTimers();
+  });
+
+  test('calls onPressIn, onPress and onPressOut prop of touchable', async () => {
+    const { events, logEvent } = createEventLogger();
+    const user = userEvent.setup();
+
+    render(
+      <Pressable
+        onPress={logEvent('press')}
```

Comment:
[mid] Adding `onLongPress={logEvent('longPress')}` in each test, would allow us to make sure it's not actually invoked when not expected.

Reply:
I added `onLongPress={logEvent('longPress')}` in tests that use `logEvent`